src/overview/reducer.js

```diff
@@ -29,6 +29,9 @@ const defaultState = {
     showOnlyBookmarks: false,
     activeTagIndex: -1,
     tags: [],
```

Comment:
I think this `tags` state isn't used anymore. Can remove it and all assoc. action/reducer logic if so. May be good to add a comment on `filterPopup` to explain it's expected values -> behaviour.